TCAT/AppDelegate.swift

```diff
@@ -24,6 +24,27 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         
+        // Update shortcut items
+        var shortcutItems = [UIApplicationShortcutItem]()
+        let favorites = SearchTableViewManager.shared.retrieveRecentPlaces(for: Constants.UserDefaults.favorites)
+        for itemType in favorites {
+            switch itemType {
+            case .busStop(let busStop):
```

Comment:
seem like `.busStop` and `.placeResult` have similar code. Try putting their code into a method to reduce repeated code